### PR TITLE
chore: Add Dependabot for updates 

### PR DESCRIPTION
**Pull request by nschonni**
Dependabot will open PRs for NPM and GitHub Action dependencies.
Filter on CI so there isn't double runs when it creates the PR branches

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: Tests
 
 on:
   push:
+    branches-ignore:
+      - "dependabot/**"
```
Comment (BenoitZugmeyer) [inline .github/workflows/tests.yml:6]: Any reason to skip CI for dependabot branches?

Comment (nschonni) [reply .github/workflows/tests.yml:6]: Because they get submitted as PRs, they already get built for the PR. This avoids running the same build twice

Comment (BenoitZugmeyer) [reply .github/workflows/tests.yml:6]: Isn't it the same for other PRs?

Comment (nschonni) [reply .github/workflows/tests.yml:6]: Yes, PRs that you create would also double build. The alternate that some folks use would be:
```
push:
  branches:
    - master
```
So it only builds when the default branch pushes. I depends if you want to see CI builds while you work on your own stuff though

Comment (BenoitZugmeyer) [reply .github/workflows/tests.yml:6]: After investigation, I prefer this strategy. Thanks for pointing this out :)

**Pull request merged**